packages/core/src/components/button/abstractButton.tsx

```diff
@@ -26,6 +26,8 @@ export interface IButtonProps extends IActionProps {
      * Text alignment within button. By default, icons and text will be centered within the button.
      * Passing this prop will cause the text container to fill the button and align the text within that
      * to the appropriate side. `icon` and `rightIcon` will be pushed to either side.
+     *
+     * The `Alignment` enum provides constants for allowed values.
```

Comment:
then why not use the enum here?

Reply:
like #2090, i want to show the actual options in the docs, not the type alias name.